rmw_freertps_cpp/CMakeLists.txt

```diff
@@ -21,8 +21,7 @@ ament_export_definitions(${freertps_DEFINITIONS})
 ament_export_include_directories(${freertps_INCLUDE_DIRS})
 
 link_directories(${freertps_LIBRARY_DIRS})
-add_library(rmw_freertps_cpp src/functions.cpp)
-#target_link_libraries(rmw_freertps_cpp rosidl_typesupport_freertps_cpp)
+add_library(rmw_freertps_cpp SHARED src/functions.cpp)
```

Comment:
I agree; I was going to write a comment on a past thread about this. I could look into making the necessary changes to ament.